src/ibmca-opensslconfig.in

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/perl
```

Comment:
Can you please rename ibmca-opensslconfig.in to something that includes the term 'engine' ? 

Sooner or later we will have an IBMCA provider. The provider will most likely also have a similar script to generate the OpenSSL config, but the resulting OpenSSL config is different. Thus I would request that we name the script for engines in a way that it has the term 'engine' in its name, and that we can easily add a second script for the provider without having a name clash, and without the need of renaming the engine-script at that time.

Reply:
Suggestions?  How about `ibmcaengine-opensslconfig.in`?

Reply:
How about `ibmca-engine-opensslconfig.in` (with a dash between ibmca and engine)?

Reply:
Okay, will fix.

Reply:
Done.